checks/check_sql

```diff
@@ -38,7 +38,9 @@ def check_sql_arguments(params):
         lower = "", ""
 
     if "perfdata" in params:
-        args.append("--metrics")
+        if not params["perfdata"]:
+            params["perfdata"] = "performance_data"
+        args.append("--metrics=%s" % params["perfdata"])
```

Comment:
There are some issues here.
I don't think we can have the `not params["perfdata"]` case, if we prohibit empty strings (see below).
Changing the functions argument (params) is dangerous. I suggest
```
if (metrics:=params.get("perfdata")) is not None:
    args.append(f"--metrics={metrics}")
```

Please add a test for this in tests/unit/checks/test_check_sql.py!